bin/kuzzle

```diff
@@ -40,16 +40,16 @@ program
   .arguments('[name]')
   .description('manage plugins')
   .option('    --list', 'list currently installed plugins')
-  .option('    --install', 'if plugin [name] is provided, install it from --version, --url or --path, otherwise, (re-)install all listed plugins')
-  .option('    --remove', 'removes plugin [name] from Kuzzle')
-  .option('    --activate', 'mark the plugin as "activated" (Kuzzle ignores deactivated plugins)')
-  .option('    --deactivate', 'mark the plugin as "deactivated" (Kuzzle ignores deactivated plugins)')
-  .option('    --importConfig <file>', 'import a configuration from a file for a given plugin')
+  .option('    --install', 'install a new plugin package from npm repository if [name] if provided or from --path or --url options')
```

Comment:
Wow, ok, sorry, I'm wrong. The option (for the CLI command) is actually `packageVersion` (translated to `version` [here](https://github.com/kuzzleio/kuzzle/blob/develop/bin/commands/plugins.js#L28)). But, still, I don't think it's mandatory.